src/utiltime.cpp

```diff
@@ -30,14 +31,14 @@ void SetMockTime(int64_t nMockTimeIn)
 
 int64_t GetTimeMillis()
 {
-    return (boost::posix_time::microsec_clock::universal_time() -
-            boost::posix_time::ptime(boost::gregorian::date(1970,1,1))).total_milliseconds();
+    int64_t now = std::chrono::duration_cast<std::chrono::milliseconds>(
```

Comment:
@daira just spotted in our pairing that this doesn't actually return anything now. What is strange is that this compiled with `-Werror`, because it should be trivial for the compiler to detect that this is undefined behaviour.